### DISCUSSION TITLE
Outdent shortcut not working in bold editor

### DISCUSSION THREAD

**Discussion by IceFire23** [standardnotes/forum#2471] [General] [answered]
**Describe the bug**
I found https://docs.standardnotes.org/usage/bold-editor/ and https://github.com/standardnotes/forum/issues/858, both of which list the outdent list item shortcut as "ctrl + [". However, that does not appear to work. Similarly, indent is listed as "ctrl + ]" but the actual shortcut that works is "tab".

**To Reproduce**
1. Create a list, then type some text
2. Press enter to move to a bullet point on the next line
3. Press tab to indent bullet point
4. [Repro Step] Press "ctrl + [" to try to outdent the indented bullet point. Nothing happens.

**Expected behavior**
The indented bullet point would be outdented back to the default indentation level.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 2004

**Other information**
Tab/Shift+Tab are the shortcuts for indenting/outdenting I'm most used to and I think are the most common shortcuts for this behavior.

**Comment by JaspalSuri** [comment]
Hi @IceFire23,

I'm sorry to hear that outdenting currently doesn't work on Windows desktop app. I was able to reproduce the issue on Windows 10, but not on macOS Big Sur. Using <KBD>shift</KBD>+<KBD>tab</KBD> doesn't work for me. 

I'll be sure to create a report for this for the team to review. Thanks for bringing this to our attention!

**Comment by IceFire23** [comment]
Sure thing, thanks for the help!

**Comment by JaspalSuri** [comment]
Hi @IceFire23,

I just wanted to let you know that we'll have to put a fix for this on hold until we can provide better keyboard shortcut support to the desktop and web apps.

**Comment by IceFire23** [comment]
Hi @JaspalSuri, thanks for the information. Do you happen to know if there's some sort of thread or task tracking the work about the improved keyboard shortcut support that I would be able to use to track progress/when the issue starts being worked on?

**Comment by JaspalSuri** [comment]
Hi @IceFire23, there currently isn't one, but the best way to follow updates that are available for testing would be to join the #beta-testers channel in our [Slack](https://standardnotes.org/slack).

**Comment by mywill** [comment]
Running into this same issue now with the outdent keybind not working.   Has any movement happened on this?

**Comment by myreli** [comment, marked as answer]
The Bold Editor has been [deprecated](https://github.com/standardnotes/plugins) in favour of Rich Text and the [new Super Notes (beta)](https://standardnotes.com/blog/introducing-super-notes-moments-and-offline-file-access). Please experiment with Rich Text which offers the `ctrl + [` and  `ctrl + ]`.

I will close this issue as resolved, let me know if you find any other problem.
